modules/tinymce/src/plugins/template/test/ts/browser/TemplateSanityTest.ts

```diff
@@ -76,64 +77,81 @@ describe('browser.tinymce.plugins.template.TemplateSanityTest', () => {
     TinyAssertions.assertContent(editor, '<p>Tester</p>');
   });
 
-  it('TINY-9244: Sanitized html should be shown when previewing template', async () => {
-    const editor = hook.editor();
-    const unsanitizedHtml = '<img src="error" onerror="throw new Error();">';
-    addSettings({
-      templates: [{ title: 'a', description: 'b', content: unsanitizedHtml }],
-    });
-    const unsanitizedPreviewHtmlSelector = 'p > img[src="error"][onerror="throw new Error();"]';
-    const sanitizedPreviewHtmlSelector = 'p > img[src="error"][data-mce-src="error"]';
-    const assertPreviewContent = (dialogEl: SugarElement<Node>, existsSelector: string, notExistsSelector: string): void => {
-      UiFinder.findIn<HTMLIFrameElement>(dialogEl, 'iframe').fold(
-        () => assert.fail('Preview iframe not found'),
-        (iframe) => {
-          // fallback for pre-IE 8 using contentWindow.document
-          const iframeDoc = iframe.dom.contentDocument || iframe.dom.contentWindow?.document;
-          const iframeBody = SugarElement.fromDom(iframeDoc?.body as Node);
-          UiFinder.exists(iframeBody, existsSelector);
-          UiFinder.notExists(iframeBody, notExistsSelector);
-        }
-      );
+  context('Previewing unparsed content', () => {
+    const unparsedHtml = '<img src="error" onerror="throw new Error();">';
+    const unparsedPreviewHtmlSelector = 'p > img[src="error"][onerror="throw new Error();"]';
+    const parsedPreviewHtmlSelector = 'p > img[src="error"][data-mce-src="error"]';
+
+    const pPreviewAndAssertNoUnparsedContent = async (editor: Editor): Promise<void> => {
+      const assertNoUnparsedContent = (dialogEl: SugarElement<Node>): void => {
+        UiFinder.findIn<HTMLIFrameElement>(dialogEl, 'iframe').fold(
+          () => assert.fail('Preview iframe not found'),
+          (iframe) => {
+            // fallback for pre-IE 8 using contentWindow.document
```

Comment:
do we still need this fallback?

Reply:
We likely don't, this was left over from existing code. I'll remove it.